internal/cltest/fixtures.go

```diff
@@ -216,15 +216,15 @@ func NewRunLog(jobID string, addr common.Address, blk int, json string) ethtypes
 	}
 }
 
-func NewSpecAndRunLog(addr common.Address, blk int, json string) ethtypes.Log {
+func NewSpecAndRunLog(addr common.Address, blk int, json string, link *big.Int) ethtypes.Log {
```

Comment:
Should this be of type `Link`? It is for fixtures so I understand the leniency.
```go
// Link contains a field to represent the smallest units of LINK
type Link big.Int
```

Reply:
It's less convenient for tests and it never gets directly compared to link, it is immediately converted into a hex string.